docs/use-cases/configure.md

```diff
@@ -3,6 +3,68 @@ title: "Build a simple smart machine"
+      <b>3. Configure your components</b><br><br>
+      <p>Each piece of your smart machine that is controlled by a computer is called a <i>component</i>. For example, if your smart machine includes a Raspberry Pi, a motor, and a camera, each of those is a component.</p>
+      <p>You need to <i><a href="/build/configure/">configure</a></i> your machine so that <code>viam-server</code> can interact with its hardware. Use the configuration builder tool in the Viam app to create a file that describes what hardware you are using and how it is connected.
+      For example, if you have a DC motor, follow the <a href="/components/motor/gpio/">corresponding configuration instructions</a> to tell the software which pins it is connected to.</p>
+    </th>
+  </tr>
+  <tr>
+    <th>{{<gif webm_src="/tutorials/lazy-susan/control-dcmotor.webm" mp4_src="/tutorials/lazy-susan/control-dcmotor.mp4" alt="The Viam app Control tab with a control panel for each component. The panel for a DC motor is clicked, expanding to show power controls." max-width="400px" class="fill alignleft">}}
+      <b>4. Test your components</b><br><br>
+      <p>When you configure a component, a remote control panel is generated for it in the <b>Control</b> tab of the Viam app. Here, you can drive motors at different speeds, view your camera feed, see sensor readings, and generally test the basic functionality of your robot before you've even written any code.
+      </p>
+    </th>
+  </tr>
+  <tr>
+    <th>{{<imgproc src="/ml/collect.svg" class="fill alignright" style="max-width: 220px"  declaredimensions=true alt="Services">}}
+      <b>5. Configure services</b><br><br>
+      <p>Services are built-in Viam software packages that add high-level functionality to your smart machine like computer vision or motion planning.
+      If you want to use any services, see their <a href="/services/">documentation</a> for configuration and usage information.
+      If you are making a simple machine that doesn't use services, you can skip this step!
+      </p>
+    </th>
+  </tr>
+  <tr>
+    <th>{{<imgproc src="/ml/configure.svg" class="fill alignleft" style="max-width: 210px"  declaredimensions=true alt="Services">}}
+      <b>6. Do more with code</b><br><br>
+      <p>Write a program to control your smart machine using the programming language of your choice. Viam has <a href="/sdks/">SDKs</a> for Python, Golang, C++, TypeScript and Flutter.</p>
+      <p>The easiest way to get started is to copy the auto-generated boilerplate code from your machine's <b>Code sample</b> tab in the Viam app. You can run this code directly on the machine or from a separate computer; it will connect to the machine using API keys.</p>
+    </th>
+  </tr>
+</table>
+
+## Next steps
+
+{{< cards >}}
+{{% card link="/get-started/try-viam/" %}}
+{{% card link="/tutorials/get-started/lazy-susan/" %}}
```

Comment:
Thought this or mouse mover would be a good example of a very simple machine that moves things instead of just blinking the led--what's your thought? Is there another one you'd recommend replacing this with?

Reply:
I just think 4 is a bit much so that's why I tried to eliminate one but ok to keep 4. I don't feel that strongly

Reply:
okay i like the 4 because then they make a nice square instead of an upside-down L shape (depending on screen width) and they're all different so I think I'll leave as-is if that's ok!